### DISCUSSION TITLE
glTF/PbrMetallicRoughness render pink

### DISCUSSION THREAD

**Discussion by rycsim** [atteneder/glTFast#373] [Q&A] [answered]
I'm testing gltF-demo-2020.3hdrp. When I use the shader "glTF/PbrMetallicRoughness" the runtime export work well but unity render it "pink" . Am I doing something bad ? Thanks for advance for the help

![Capture](https://user-images.githubusercontent.com/1567978/159269533-36a36a24-a54d-46b7-bc15-1ba1b9368711.JPG)

**Comment by atteneder** [comment, marked as answer]
This is not a discussion, this is a error report and thus I created [an issue](https://github.com/atteneder/glTFast/issues/377)
